reanalysis/comparison.py

```diff
@@ -128,18 +139,108 @@ def common_format_from_results(results_dict: Dict[str, Any]) -> CommonDict:
     return sample_dict
 
 
-def main(results: str):
+def common_format_from_seqr(seqr: str, probands: List[str]) -> CommonDict:
+    """
+    identify the most likely proband for each row, and create a variant object for them
+
+    :param seqr:
+    :param probands:
+    :return:
+    """
+
+    assert seqr.endswith('.tsv'), f'process requires a TSV format export'
+    sample_dict: CommonDict = defaultdict(list)
+
+    with AnyPath(seqr).open() as handle:
+        seqr_parser = DictReader(handle, delimiter='\t')
+
+        sample_cols = [
+            sample_field
+            for sample_field in seqr_parser.fieldnames
+            if SAMPLE_NUM_RE.match(sample_field)
+        ]
+
+        for entry in seqr_parser:
+
+            # get all valid tags
+            tags = [
+                Confidence(tag)
+                for tag in entry['tags'].split('|')
+                if tag in VALID_VALUES
+            ]
+
+            # no relevant tags, not interested...
+            if len(tags) == 0:
+                continue
+
+            # seqr export currently has no notion of 'proband', so find the
+            # most likely candidate
+            for index, value in enumerate(sample_cols, 1):
+                if (
+                    entry[value] in probands
+                    and int(entry[SAMPLE_ALT_TEMPLATE.format(index)]) > 0
+                ):
+                    sample_dict[entry[value]].append(
+                        CommonFormatResult(
+                            entry['chrom'],
+                            int(entry['pos']),
+                            entry['ref'],
+                            entry['alt'],
+                            confidence=tags,
+                        )
+                    )
+
+    logging.info(f'Variants from Seqr digest: {sample_dict}')
+
+    return sample_dict
+
+
+def find_probands(pedigree: Ped) -> List[str]:
```

Comment:
Nitpick: By definition, a family can only have a single proband and it is not always a child.  Most of the time in our cohorts this will be a distinction without a difference, but I would still rather not use this language.

Reply:
updated - this method is retitled `find_affected_samples`, and picks up all affected participants in the pedigree